code/modules/surgery/organs/organ_external.dm

```diff
@@ -487,14 +487,43 @@ Note that amputating the affected organ does in fact remove the infection from t
 		//Open wounds can become infected
 		if(owner.germ_level > W.germ_level && W.infection_check())
 			W.germ_level++
-
+	for(var/obj/item/organ/external/O in owner)
+		//incisions make it easier to get an infection
+		if(O.open)
+			spread_germs_by_incision(O)
 	if(antibiotics < 5)
 		for(var/datum/wound/W in wounds)
 			//Infected wounds raise the organ's germ level
 			if(W.germ_level > germ_level)
 				germ_level++
 				break	//limit increase to a maximum of one per second
 
+/proc/spread_germs_by_incision(obj/item/organ/external/E)
+	if(!istype(E))
+		return
+
+	if(!(E.status & ORGAN_ROBOT))
+		var/germ_count = 0
+		var/area/room = get_area(E)
+		for(var/mob/living/carbon/human/H in room)
+			if(!(NO_BREATH in H.mutations) || !(H.species.flags & NO_BREATH))
+				if(H.wear_mask)
```

Comment:
Shouldn't wearing a mask prevent all germs. Yes, I know "well, it's just simple fibers", but this is sci-fi----and we shouldn't be punishing players for performing a surgery correctly.

This is the quintessential "punish players less, but still punish them for making the right choice" design, and I don't think it works here.